dao/src/main/java/org/thingsboard/server/dao/model/sql/MobileAppSettingsEntity.java

```diff
@@ -0,0 +1,94 @@
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.thingsboard.common.util.JacksonUtil;
+import org.thingsboard.server.common.data.id.TenantId;
+import org.thingsboard.server.common.data.mobile.AndroidConfig;
+import org.thingsboard.server.common.data.mobile.IosConfig;
+import org.thingsboard.server.common.data.mobile.MobileAppSettings;
+import org.thingsboard.server.common.data.mobile.QRCodeConfig;
+import org.thingsboard.server.dao.model.ModelConstants;
+import org.thingsboard.server.dao.model.ToData;
+import org.thingsboard.server.dao.util.mapping.JsonConverter;
+
+import java.io.Serializable;
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
+@Entity
+@Table(name = ModelConstants.MOBILE_APP_SETTINGS_TABLE_NAME)
+public class MobileAppSettingsEntity implements ToData<MobileAppSettings>, Serializable {
+
+    @Id
+    @Column(name = ModelConstants.TENANT_ID_COLUMN, columnDefinition = "uuid")
+    protected UUID tenantId;
+
+    @Column(name = ModelConstants.MOBILE_APP_SETTINGS_USE_DEFAULT_APP_PROPERTY)
+    private boolean useDefaultApp;
+
+    @Convert(converter = JsonConverter.class)
+    @Column(name = ModelConstants.MOBILE_APP_SETTINGS_ANDROID_CONFIG_PROPERTY)
+    private JsonNode androidConfig;
+
+    @Convert(converter = JsonConverter.class)
+    @Column(name = ModelConstants.MOBILE_APP_IOS_CONFIG_PROPERTY)
+    private JsonNode iosConfig;
+
+    @Convert(converter = JsonConverter.class)
+    @Column(name = ModelConstants.MOBILE_APP_QR_CODE_CONFIG_PROPERTY)
+    private JsonNode qrCodeConfig;
+
+    public MobileAppSettingsEntity(MobileAppSettings mobileAppSettings) {
+        this.tenantId = mobileAppSettings.getTenantId().getId();
+        this.useDefaultApp = mobileAppSettings.isUseDefaultApp();
+        if (mobileAppSettings.getAndroidConfig() != null) {
+            this.androidConfig = JacksonUtil.valueToTree(mobileAppSettings.getAndroidConfig());
+        }
+        if (mobileAppSettings.getIosConfig() != null) {
+            this.iosConfig = JacksonUtil.valueToTree(mobileAppSettings.getIosConfig());
+        }
+        if (mobileAppSettings.getQrCodeConfig() != null) {
+            this.qrCodeConfig = JacksonUtil.valueToTree(mobileAppSettings.getQrCodeConfig());
+        }
+   }
+
+    @Override
+    public MobileAppSettings toData() {
+        MobileAppSettings mobileAppSettings = new MobileAppSettings();
+        mobileAppSettings.setTenantId(TenantId.fromUUID(tenantId));
+        mobileAppSettings.setUseDefaultApp(useDefaultApp);
+        if (qrCodeConfig != null) {
+            mobileAppSettings.setAndroidConfig(JacksonUtil.convertValue(androidConfig, AndroidConfig.class));
+        }
+        if (qrCodeConfig != null) {
+            mobileAppSettings.setIosConfig(JacksonUtil.convertValue(iosConfig, IosConfig.class));
+        }
+        if (qrCodeConfig != null) {
+            mobileAppSettings.setQrCodeConfig(JacksonUtil.convertValue(qrCodeConfig, QRCodeConfig.class));
+        }
```

Comment:
After adding id column and extending BaseSqlEntity, use parent's `toJson` and `fromJson`